### PR TITLE
Add use in TypeScript instruction

### PR DESCRIPTION
**Pull request by yesmeck**

### CODE REVIEW

File: docs/react/use-in-typescript.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+$ yarn create react-app antd-demo-ts --scripts-version=react-scripts-ts
+```
+
+如果你使用的是 npm（接下来我们都会用 yarn 作为例子，如果你习惯用 npm 也是一样的）。
+
+```bash
+$ npm install -g create-react-app
+$ create-react-app antd-demots --scripts-version=react-scripts-ts
```
Comment (afc163) [inline docs/react/use-in-typescript.zh-CN.md]: antd-demo-ts

**Comment by afc163** [comment]
和这个有啥区别？
https://github.com/wmonk/create-react-app-typescript

File: docs/react/use-in-typescript.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+@import '~antd/dist/antd.css';
+
+.App {
+  text-align: center;
+}
+
+...
+```
+
+这个时候你可能会碰到下面类似的错误，这是因为 antd 的类型定义还不完善（[帮助我们完善定义](https://github.com/ant-design/ant-design/issues/5627)）。
```
Comment (afc163) [inline docs/react/use-in-typescript.zh-CN.md]: 👍，丧心病狂的文档细节

**Comment by yesmeck** [comment]
> 和这个有啥区别？
> https://github.com/wmonk/create-react-app-typescript

这个就是 `react-scripts-ts`

**Comment by abenhamdine** [comment]
FWIW, I found out an alternative way to use typescript with antd, with https://github.com/lwd-technology/react-app-rewire-typescript, and there's no need to eject : 

package.json
```
"scripts": {
		"start": "react-app-rewired start",
		"build": "react-app-rewired build",
		"test": "react-app-rewired test --env=jsdom",
		"eject": "react-app-rewired eject"
	},
```

config-override.js
```ts
const { injectBabelPlugin } = require('react-app-rewired')
const rewireLess = require('react-app-rewire-less')
const rewireTypescript = require('react-app-rewire-typescript')

module.exports = function override(config, env) {

	config = rewireTypescript(config, env)

	const path = require('path')
	// For import with absolute path
	config.resolve.modules = [path.resolve('src')].concat(config.resolve.modules)

	config = injectBabelPlugin(['import', { libraryName: 'antd', style: true }], config)  // change importing css to less

	const options = {
		// We use modifyVars option of less-loader here, you can see a orange button rendered on the page after reboot start server.
		modifyVars: {
			"@primary-color": "#f48b31",
		},
	}
	config = rewireLess.withLoaderOptions(options)(config, env)

	return config
}
```

**Comment by yesmeck** [comment]
By using `react-scripts-ts`,  there's no need to eject too.

**Comment by abenhamdine** [comment]
> By using react-scripts-ts, there's no need to eject too.

Yes. It was just about suggesting an alternative way with same advantages (no eject) and perhaps more robust, because it don't depends on https://github.com/wmonk/create-react-app-typescript, but directly on creat-react-app, so the user can benefit immediatly of last versions of cra without waiting for updates by wmonk.

**Comment by yesmeck** [comment]
Personally，I want keep this instruction as simple as possible, `react-scripts-ts` setup a whole TypeScript development env for you, includes `tsconfig.json`, `tsconfig.test.json` and tslint.  I'll mention `react-app-rewire-typescript` in this doc, thank you.

**Comment by qinyang1980** [comment]
(create react app + typescript + antd) without eject and without using Babel.

https://github.com/qinyang1980/create-react-antd-app.git

**Pull request merged**